src/scripts/installOED.sh

```diff
@@ -145,7 +145,7 @@ else
 	# Create a user
 	set -e
 	if [ "$production" == "no" ] && [ ! "$OED_PRODUCTION" == "yes" ]; then
-		npm run createUser -- test@example.com password
+		npm run createUser -- testUser password
```

Comment:
First, given you had to edit this in two places it probably should have been a variable from the beginning. Second, while I like having shorter username without the special symbols I have mixed feeling about changing it. A lot of people expect the old email and this may throw them off. I'm inclined to either leave it alone or insert a second admin user with the new name. What do you thinkg?

Reply:
Up to you, I just thought since we were moving away from email, it would be better to use a username. I am good with whatever you prefer. I'll change it back and leave it how it was for now. Let me know and I can have it add both if you'd prefer.

Reply:
Lets put in user called test with the same password to start to transition. Then leave a TODO to remove the old one in the future.

Reply:
Will do.